Drop-In/src/main/java/com/braintreepayments/api/BottomSheetFragment.java

```diff
@@ -51,7 +51,14 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
 
         View view = inflater.inflate(R.layout.bt_fragment_bottom_sheet, container, false);
         backgroundView = view.findViewById(R.id.background);
+
         viewPager = view.findViewById(R.id.view_pager);
+        // TODO: investigate view pager saveInstanceState restoration
+        viewPager.setSaveEnabled(false);
+
+        // it's best to call bind here before any live data / fragment result observers are registered
```

Comment:
Just curious, do you have a TLDR on why it's better to call bind here?

Reply:
Yeah we observe the `ViewModel` for changes in `onCreate`, and some of the `LiveData` in the view model have initial values. When there is initial data, the observer callbacks fire as soon as we call `observe()`, so we need to instantiate `bottomSheetPresenter` early to make sure we don't get null pointer exceptions that cause the app to crash.

Reply:
The true TLDR is "because Java" lol.